modules/Emails/EmailUI.php

```diff
@@ -422,6 +422,24 @@ public function generateComposePackageForQuickCreateFromComposeUrl($emailLinkUrl
         return $this->generateComposePackageForQuickCreate($a_composeData, $emailLinkUrl, $lazyLoad);
     }
 
+    private function createEmailLink($module_name, $record_id, $name, $addr, $text) {
+        global $current_user;
+
+        if ($current_user->getEmailClient() == 'sugar') {
+            return '<a class="email-link"'
+                . ' onclick="$(document).openComposeViewModal(this);"'
+                . ' data-module="' . $module_name
+                . '" data-record-id="' . $record_id
+                . '" data-module-name="' . $name
+                . '" data-email-address="' . $addr  . '">'
+                . $text . '</a>';
+        } else {
```

Comment:
Using `else` is not necessary after a return statement (and I think it's bit confusing). Also it improves the code complexity. So I don't think it's needed here.